modules/lb-vshn-hiera/main.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  instance_fqdns = formatlist("%s.${var.node_name_suffix}", var.lb_names)
+
+  lb_count = length(var.lb_names)
+
+  content = templatefile(
+    "${path.module}/templates/hieradata.yaml.tmpl",
+    {
+      "cluster_id"   = var.cluster_id
+      "api_vip"      = cidrhost(var.api_vip_network, 0)
+      "router_vip"   = cidrhost(var.router_vip_network, 0)
+      "api_secret"   = var.lb_cloudscale_api_secret
```

Comment:
The Ursula/Floaty secret handling will need to be made more generic, if we want to reuse this module for Exoscale, but that can wait.